encryptor/queryDataEncryptor.go

```diff
@@ -316,6 +316,12 @@ func (encryptor *QueryDataEncryptor) OnQuery(query base.OnQueryObject) (base.OnQ
 	return query, false, nil
 }
 
+// ErrInvalidPlacholder is returned when Acra cannot parse SQL placeholder expression.
+var ErrInvalidPlacholder = errors.New("invalid placeholder value")
```

Comment:
ErrInvalidPlac**e**holder

Reply:
I guess I'm in a hurry and have an affinity for typos today :joy: